### PR TITLE
Qp nil clean

### PR DESCRIPTION
**Pull request by PatLittle**
QPnotes nil report

### CODE REVIEW

File: bin/pd/Makefile [hunk 1]
```diff
@@ -206,23 +206,34 @@ $(workdir)/filtered/briefingt.csv: $(workdir)/briefingt.csv
...
 .PHONY: upload-qpnotes
-upload-qpnotes: $(workdir)/filtered/qpnotes.csv
+upload-qpnotes: $(workdir)/filtered/qpnotes.csv $(workdir)/filtered/qpnotes-nil.csv
 	@$(echo_date) Uploading Question Period Notes...
 	@$(ckanapi) action resource_patch -c $(registry_ini) \
 	id=c55a2862-7ec4-462c-a844-22acab664812 upload@"$(workdir)/filtered/qpnotes.csv" \
 	> /dev/null
+	# @$(ckanapi) action resource_patch -c $(registry_ini) \
+	# id=TBD upload@"$(workdir)/filtered/qpnotes-nil.csv" \
+	> /dev/null
```
Comment (wardi) [inline bin/pd/Makefile:216]:
```suggestion
	# > /dev/null
```

File: ckanext/canada/tables/qpnotes.yaml [hunk 2]
```diff
@@ -357,3 +357,123 @@ excel_column_heading_style:
   PatternFill:
     patternType: solid
     fgColor: FFEFEFEF
+
+- title: Proactive Publication - Question Period Notes Nothing to Report
```
Comment (wardi) [inline ckanext/canada/tables/qpnotes.yaml:361]: We need a corresponding entry in the .po for this

**Comment by wardi** [comment]
otherwise looks good

File: ckanext/canada/tables/qpnotes.yaml (hunk 2 above)
Comment (PatLittle) [reply ckanext/canada/tables/qpnotes.yaml:348]: I think I got that figured out now

**Review by wardi — APPROVED** [review]

**Pull request merged**